Sources/StreamChat/Workers/Background/NewUserQueryUpdater.swift

```diff
@@ -14,59 +14,56 @@ import CoreData
 ///     corresponding queries to the user.
 final class NewUserQueryUpdater: Worker {
```

Comment:
Any reason why we have only Updaters, but they are actually in a "Workers" folder, and each updater is a "Worker". Can't we just rename "Worker" to "Updater"? @b-onc

Reply:
No clue. I cannot answer that question, sorry 😐